targets/TARGET_Maxim/TARGET_MAX32630/serial_api.c

```diff
@@ -205,7 +213,7 @@ void serial_irq_handler(serial_t *obj, uart_irq_handler handler, uint32_t id)
 void serial_irq_set(serial_t *obj, SerialIrq irq, uint32_t enable)
 {
     MBED_ASSERT(obj->index < MXC_CFG_UART_INSTANCES);
-    objs[obj->index] = obj;
+    // objs[obj->index] = obj;
```

Comment:
why is the line just commented ?

Reply:
it could be removed.  It was a copy over from the MAX32620C implementation serial_api.c.

Reply:
Please remove unused code